vcpkg/commands/add-version.md

```diff
@@ -0,0 +1,52 @@
+---
+title: vcpkg x-add-version
+description: Command line reference for the vcpkg x-add-version command. Update version database of vcpkg ports.
```

Comment:
```suggestion
description: Reference for the vcpkg x-add-version command. Update version database of vcpkg ports.
```

@AugP and @vicroms 

We decided to remove the "Command line" at the beginning of these descriptions to make it shorter, reduce the repetition of the "command' word and not having to deal with ["command-line" vs "command line"](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/command-line).